### DISCUSSION TITLE
SP Page Builder

### DISCUSSION THREAD

**Discussion by starshiphamburger** [Quantum-Manager/tracker#76] [Q&A]
Hi Support, 

I have been using Quantum for a while and I must say VERY NICE JOB. Works amazingly fast and super. However, as an SP Page Builder user (most of my sites), I have noticed or maybe it's just me, no integration even if your changelog says this has been fixed.  See file attached.

![CleanShot 2021-04-07 at 15 46 44@2x](https://user-images.githubusercontent.com/82095499/113861803-a8668600-97b8-11eb-98c4-12f8b254de52.jpg)

I am using SP Page Builder Pro Version: 3.7.9. Is this to do with the actual editor (currently Tiny MCE) or is there a specific editor for this ?
Thanks again!! and amazing work guys

**Comment by ssnobben** [comment]
Is this working now with latest SP Pagebuilder Pro too?

**Reply by starshiphamburger** [reply]
Hi bro, I have tested on SPPB V5 Pro and it does not work. V5 uses its own media manager. However, SPPB V5 has a 'system editor' that allows you to access JCE Pro (or other editors) but still no access to Quantum!

**Comment by ssnobben** [comment]
> Hi bro, I have tested on SPPB V5 Pro and it does not work. V5 uses its own media manager. However, SPPB V5 has a 'system editor' that allows you to access JCE Pro (or other editors) but still no access to Quantum!

Ok thks for update. 

hmm for me its useless then.. 

Bcs I wanted to use it with latest SP Pagebuilder Pro php 8.2.+ and not working.. a pity.

cheers!
